cscs-checks/mch/collectives_halo.py

```diff
@@ -40,6 +40,16 @@ def __init__(self, variant, bench_reference):
                 '-DCUDA_COMPUTE_CAPABILITY="sm_60"'
             ]
             self.build_system.max_concurrency = 8
+        else:
+            self.num_tasks = 4
+            self.num_gpus_per_node = 1
+            self.num_tasks_per_node = 1
+            self.variables['MPICH_RDMA_ENABLED_CUDA'] = '1'
```

Comment:
If this and the next environment variables are optional don't use them for the generic case.

Reply:
Exactly. The generic version should be bare bones. You don't know anything about the system and the will need adaptations in any case. The important thing is for the test not to crash.